content/docs/components/central-dashboard.md

```diff
@@ -0,0 +1,11 @@
++++
+title = "Central Dashboard"
+description = "All about central dashboard in the Kubeflow Ecosystem"
```

Comment:
It's not really about the ecosystem, but more about the Kubeflow system itself. It'd also be good to use words different from those in the title, so that people who don't understand the title have a different view into what this page is about. How about this for the description:

```
An overview of the Kubeflow user interface (UI)
```